src/plugins/explorer/sync-transactions.js

```diff
@@ -173,14 +198,7 @@ function createSyncer (config, eventBus, web3, queue, eventsRegistry, indexer) {
           resyncing = true
           shallResync = false
           // eslint-disable-next-line promise/catch-or-return
-          contract.getPastEvents(
-            eventName,
-            { fromBlock: bestSyncBlock, filter }
-          )
-            .then(function (events) {
-              debug(`${events.length} past ${eventName} events retrieved`)
-              events.forEach(queue.addEvent(address, metaParser))
-            })
+          getPastEventsWithChunks({ address, contract, eventName, fromBlock: bestSyncBlock, toBlock: undefined, filter, metaParser })
```

Comment:
Check the code format here. Didn't the linter/formatter catch this?

Reply:
`toBlock` should not be `undefined` as it would break inside `getPastEventsWithChunks`. Let's pass `toBlock: number` as that is the number of the block that was just received.

Reply:
Fixed here https://github.com/autonomoussoftware/metronome-wallet-core/pull/32/commits/d210679451505871c1f651ee8771624fe4ceb25d

While it worked (because the default value from the library was `latest`), I think this will work better as not to rely on default values. Good suggestion!